### PR TITLE
Initialized express BE

### LINKED ISSUE #8: Setup boilerplate BE (npx init)

### PR DESCRIPTION
**Pull request by AlvinCYLo**

### CODE REVIEW

**Review by austincho — APPROVED** [review]
looks good to me

File: backend/views/error.jade [hunk 1]
```diff
@@ -0,0 +1,6 @@
+extends layout
+
+block content
+  h1= message
+  h2= error.status
+  pre #{error.stack}
```
Comment (valarao) [inline backend/views/error.jade:6]: Out of curiosity, can you explain what the `.jade` files are?

**Review by valarao — APPROVED** [review]
LGTM, left a comment for something I wasn't too sure about

File: backend/views/error.jade (hunk 1 above)
Comment (AlvinCYLo) [reply backend/views/error.jade:6]: not too sure, these were automatically generated by express generator. Looks like they're templates files that create HTML files

Comment (valarao) [reply backend/views/error.jade:6]: Ah makes sense, thanks!

**Pull request merged**